### PR TITLE
Use hbsStore for kyc in holofuel and clean-up clientStore.

### PR DESCRIPTION
**Pull request by Paterick**
- Pull HPOS service call logic from useHolochain store and move to services folder
- useClient store takes a loadKycLevel function that fetches the kyc level from hbsStore or HPOS
- remove isHPOSServed from useHolochain store
- remove unused hposHolochain calls from useHolochain store

### CODE REVIEW

File: src/stores/useClientStore.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { inspect } from 'util'
 import { defineStore } from 'pinia'
 import { encodeAgentId } from '../utils/agent'
 
-const makeUseClientStore = ({ useInterfaceStore, onInit }) => defineStore('client', {
+const makeUseClientStore = ({ useInterfaceStore, onInit, loadKycLevel }) => defineStore('client', {
```
Comment (robbiecarlton) [inline src/stores/useClientStore.js]:
```suggestion
const makeUseClientStore = ({ useInterfaceStore, onInit, fetchKycLevel }) => defineStore('client', {
```

I'd call this fetch, with the distinction being "fetch" gets something remotely, "get" gets something locally, and "load" gets something remotely and stores it locally

File: src/stores/useHolochainStore.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 import { defineStore } from 'pinia'
 import { presentHcSignal, listify } from '../utils'
 import useIsLoadingStore from './useIsLoadingStore'
 import useSignalStore from './useSignalStore'
 import { kycLevel2 } from '../services/hbs'
+import { hposHolochainCall } from '../services/hpos'
 
 const HC_APP_TIMEOUT = 35_000
 
-const makeUseHolochainStore = ({ installed_app_id, app_ws_url, is_hpos_served, hc_admin_port }) => defineStore('holochain', {
```
Comment (robbiecarlton) [inline src/stores/useHolochainStore.js:12]: yay

File: src/stores/useHolochainStore.js [hunk 3]
```diff
@@ -135,60 +123,8 @@ const makeUseHolochainStore = ({ installed_app_id, app_ws_url, is_hpos_served, h
...
-          throw new Error(`No case in hposCall for ${method} method`)
-        }
-    },    
     async loadAgentKycLevel(_, __) {
-      if( !is_hpos_served) {
-        return null // raw holochain doesn't have a mechanism for retrieving agent's kyc
-      }
-      
-      const kycLevel = await this.hposHolochainCall({path: 'kyc', headers: {}, params: {}, method: 'get'})
+      const kycLevel = await hposHolochainCall({path: 'kyc', headers: {}, params: {}, method: 'get'})
```
Comment (robbiecarlton) [inline src/stores/useHolochainStore.js:127]: I know the name of this function is not part of this PR, but by the same logic as above, this function would be called `fetchAgentKycLevel`

**Review by robbiecarlton — APPROVED** [review]
Non blocking name change suggestions, otherwise looks good

File: src/stores/useHolochainStore.js (hunk 3 above)
Comment (Paterick) [reply src/stores/useHolochainStore.js:127]: renamed

File: src/stores/useClientStore.js (hunk 1 above)
Comment (Paterick) [reply src/stores/useClientStore.js]: renamed

File: src/stores/useHolochainStore.js (hunk 2 above)
Comment (Paterick) [reply src/stores/useHolochainStore.js:12]: renamed

File: src/stores/useClientStore.js (hunk 1 above)
Comment (Paterick) [reply src/stores/useClientStore.js]: renamed.
